lib/Nitrapi/Services/Gameservers/Games/Minecraft.php

```diff
@@ -59,11 +59,11 @@ public function getOverviewMapLog() {
      * @param $
      * @return bool
      */
-    public function setOverviewMap($enabled = false, $signs = true, $ipsonly = null, $reset = false) {
+    public function setOverviewMap($enabled = false, $version = 'pre_1.13', $ipsonly = null, $reset = false) {
```

Comment:
Because this is lib is live for some time now, this can silently break existing implementationn (accidently setting $version to false|true if assumed that this is the $signs param). For that, It would be better to place new params at the end of the parameter list and leave the $signs params as it is and ignore it. This is ugly (carrying old stuff which is not needed), but this way, nothing breaks. On the other hand, the WI would be the only place this part of the PHP lib is ever used, we can live with that.